compiler-rt/lib/hwasan/hwasan_report.cpp

```diff
@@ -404,12 +404,14 @@ class BaseReport {
         stack_allocations[stack_allocations_count++].CopyFrom(t);
       }
     });
+
+    FindBufferOverflowCandidate();
```

Comment:
I think this would be clearer if `FindBufferOverflowCandidate` returned the struct, and this would read `candidate = FindBufferOverflowCandidate()`.

Reply:
done